packages/atomic/src/components/facets/atomic-numeric-facet/atomic-numeric-facet.tsx

```diff
@@ -216,21 +226,55 @@ export class AtomicNumericFacet
     this.dependenciesManager?.stopWatching();
   }
 
-  private initializeFacet() {
+  private initializeFacets() {
```

Comment:
Unfortunately, I think having two controllers of the same type add a bit too much complexity. Until I actually read what the original issue was about, I had trouble figuring out why there was two controllers.

I think that either we should [extend a custom controller](https://docs.coveo.com/en/headless/latest/usage/extend-headless-controllers/) internal to Atomic, or build that functionality in Headless somehow.

Reply:
In the end there is a need for 2 facet requests, 2 controllers is the way to go. Having a controller registering 2 facets is also pretty complex